earl/src/main/java/com/einmalfel/earl/RSSItem.java

```diff
@@ -181,6 +195,10 @@ public String getDescription() {
     return null;
   }
 
+  public String getContentEncoded() {
```

Comment:
I kind of understood the paradigm you're following. However, I couldn't find a standard for having `content:encoded` for Atom feeds, so that's why I stuck this method in `RSSItem`. Got a better placement?